axelrod/tests/strategies/test_punisher.py

```diff
@@ -22,30 +22,37 @@ class TestPunisher(TestPlayer):
 
     def test_init(self):
         """Tests for the __init__ method."""
-        P1 = axelrod.Punisher()
-        self.assertEqual(P1.history, [])
-        self.assertEqual(P1.mem_length, 1)
-        self.assertEqual(P1.grudged, False)
-        self.assertEqual(P1.grudge_memory, 1)
+        player = axelrod.Punisher()
+        self.assertEqual(player.history, [])
```

Comment:
This line is probably not necessary, or is better as a generic player test in `test_player.py`.